soldeer.lock

```diff
@@ -5,6 +5,13 @@ url = "https://soldeer-revisions.s3.amazonaws.com/@openzeppelin-contracts/5_4_0_
 checksum = "3dd38f17610dba4602bd008ee2cb551e51e97d7b4ce04e1ffdf853da832942fa"
 integrity = "4eb0ef219d10ab5a5708adaae1132dc0a93ab6a193378486e842d606c989ff3e"
 
+[[dependencies]]
+name = "bls-solidity"
+version = "0.1.0"
+url = "https://github.com/randa-mu/bls-solidity/archive/refs/heads/main.zip"
+checksum = "8c661d709c4bfb3abae3e1c9d0d7069278c1538af152546e0e0bc96a1fb274b2"
+integrity = "a2a33599cc0c7d336dfc9501490bc5cfef581e67934b74ac8457e53f9179f887"
```

Comment:
Since the url is referencing main, this will break whenever we push. We should use the tag

Reply:
Thanks for spotting that @bbjubjub2494 
It's been uninstalled and updated with `forge soldeer install bls-solidity~0.1.0 https://github.com/randa-mu/bls-solidity/archive/refs/tags/v0.1.0.zip`